### PR TITLE
docs(operators): add documentation for delayWhen

### PR DESCRIPTION
**Pull request by luillyfe**
- Added live example on stackblizt

### CODE REVIEW

**Comment by JWO719** [comment]
Thanks for the pr. Travis is broken because of your commit msg but I am not sure what's the problem. I would suggest the problem occurs becuase of the '#129'. Could you try to fix this. Thank you!

**Comment by ladyleet** [comment]
thank you @luillyfe for contributing! :)

File: src/operator-docs/utility/delayWhen.ts [hunk 1]
```diff
@@ -1,6 +1,51 @@
...
+      </p>
+    `
+  },
+  examples: [
+    {
+      name: 'It delays the emission of items ',
+      code: '',
+      externalLink: {
+        platform: 'JSBin',
+        url: 'https://stackblitz.com/edit/delaywhen?file=index.ts'
```
Comment (JWO719) [inline src/operator-docs/utility/delayWhen.ts]: Like that!

File: src/operator-docs/utility/delayWhen.ts [hunk 2]
```diff
@@ -1,6 +1,51 @@
...
+      <p>
+        If the second argument is applied, subscriptionDelay (an Observable) it delays the subscription
+        once it emmits any value.
+      </p>
+    `
+  },
+  examples: [
+    {
+      name: 'It delays the emission of items ',
+      code: '',
```
Comment (JWO719) [inline src/operator-docs/utility/delayWhen.ts]: Could you add a basic example in the code attribute

File: src/operator-docs/utility/delayWhen.ts [hunk 3]
```diff
@@ -1,6 +1,94 @@
...
+        once it emmits any value.
+      </p>
+    `
+  },
+  examples: [
+    {
+      name: 'It delays the emission of items by 1 seconds',
+      code: `
+          import { Observable } from 'rxjs/Observable';
+          import 'rxjs/add/observable/fromEvent';
```
Comment (JWO719) [inline src/operator-docs/utility/delayWhen.ts]: Hi @luillyfe . Thanks for all your PRs, really appreciate it. Could you also change these to es6 imports.

**Pull request merged**